tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module tools
 
-go 1.23.7
+go 1.24.0
+
+toolchain go1.24.2
```

Comment:
nit: Was 1.24.2 just randomly chosen? Rather than the latest atm or just omitting